languages/php/src/test/kotlin/io/vrap/codegen/languages/php/TestCodeGenerator.kt

```diff
@@ -2,54 +2,133 @@ package io.vrap.codegen.languages.php
 
 import io.vrap.codegen.languages.php.base.PhpBaseModule
 import io.vrap.codegen.languages.php.model.PhpModelModule
+import io.vrap.codegen.languages.php.test.PhpTestModule
 import io.vrap.rmf.codegen.CodeGeneratorConfig
 import io.vrap.rmf.codegen.di.ApiProvider
 import io.vrap.rmf.codegen.di.GeneratorComponent
 import io.vrap.rmf.codegen.di.GeneratorModule
+import org.apache.commons.io.IOUtils
+import org.junit.Assert
 import org.junit.Test
+import java.nio.charset.StandardCharsets
 import java.nio.file.Path
 import java.nio.file.Paths
 
-
 class TestCodeGenerator {
 
     companion object {
         private val userProvidedPath = System.getenv("TEST_RAML_FILE")
+        private val importerUserProvidedPath = System.getenv("IMPORTER_RAML_FILE")
+        private val mlUserProvidedPath = System.getenv("ML_RAML_FILE")
+        private val userProvidedOutputPath = System.getenv("OUTPUT_FOLDER")
         private val apiPath : Path = Paths.get(if (userProvidedPath == null) "../../api-spec/api.raml" else userProvidedPath)
+        private val importerPath : Path = Paths.get(if (importerUserProvidedPath == null) "../../api-spec/api.raml" else importerUserProvidedPath)
+        private val mlPath : Path = Paths.get(if (mlUserProvidedPath == null) "../../api-spec/api.raml" else mlUserProvidedPath)
+        private val outputFolder : Path = Paths.get(if (userProvidedOutputPath == null) "build/gensrc" else userProvidedOutputPath)
         val apiProvider: ApiProvider = ApiProvider(apiPath)
-        val generatorConfig = CodeGeneratorConfig(basePackageName = "")
+        val importProvider: ApiProvider = ApiProvider(importerPath)
+        val mlProvider: ApiProvider = ApiProvider(mlPath)
     }
 
-    
-    fun generatePHPModels() {
+    @Test
```

Comment:
Do these test run in our gradle build? Can we then run them without PHP? the most important thing for us as a team is that we can our build without having to install additional tools. WDYT?

Reply:
No they didn't as they were only executed when a userPath has been provided. But I changed it so, that you explicitly have to enable them.